### ISSUE TITLE
autocomplete-bibtex didn't work

### ISSUE THREAD

**Issue by m0nhawk** [apcshields/autocomplete-bibtex#37]
I install the package (version 0.6.1 and Atom 1.0.9), add the path to the BibTeX file. Type `at` (`@`) and... nothing happens.

Console have no errors and no autocomplete list appear.

**Comment by apcshields** [comment]
Andrew,

Try this:

  1. Close Atom.
  2. Modify the BibTeX file in some way. (If you are on Linux or OS X, I recommend using the `touch` command from the command line, e.g. `touch /path/to/bibtex.bib`.)
  3. Open Atom back up.
  4. In a Markdown file (the only syntax scope in which autocomplete-bibtex is configured by default to run), try typing ‘@‘ again.

Please let me know whether that works.

Thanks!
Andrew

**Comment by m0nhawk** [comment]
Ok. I thought I found that this is my BibTeX (it's generated with Mendeley).

I now get the error in console for the original one:

```
TypeError: path must be a string {stack: (...), message: "path must be a string"}
```

And how can I "debug" this to get the wrong items, so I can fix them to make it work with a package?

**Comment by apcshields** [comment]
Andrew,

That actually looks more like the error it usually throws when there is something wrong with parsing the BibTeX file _path_ in your configuration.

Make sure that the ‘Bibtex’ setting in your config is either a file name or a comma-separated list of file names. For instance, right now mine is:

```
/Users/aps/Documents/bibtex/collection.bibtex, /Users/aps/Documents/zotero.bib
```

Thanks,
Andrew

**Comment by elgitmariachi** [comment]
How can I see if there are error messages? This plugin was working and then suddenly is not anymore. I type @ and nothing happens. Blacklist is .* and I tried "touch my.bib" already. I'm using Linux.

**Comment by maecky** [comment]
@elgitmariachi @apcshields I had the same problem on win10, I submitted a [pull request](https://github.com/apcshields/autocomplete-bibtex/pull/53) which fixes this issue

**Comment by mangecoeur** [comment]
Fixed by #47
